### PR TITLE
Pasha T3

### PR DESCRIPTION
**Pull request by st-pasha**
PUBDEV-3157

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/74)

### CODE REVIEW

**Comment by abal5** [comment]
Can one of the admins verify this patch?

File: h2o-py/h2o/__init__.py [hunk 1]
```diff
@@ -25,13 +25,10 @@
 
 __version__ = "SUBST_PROJECT_VERSION"
 
-# __all__ = ['assembly', 'astfun', 'connection', 'cross_validation', 'display',
-#            'expr', 'frame', 'group_by', 'h2o',
-#            'job', 'two_dim_table', 'estimators', 'grid', 'model', 'transforms']
 
 __all__ = ("connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
-           "import_sql_select", "parse_setup", "parse_raw", "assign","deep_copy" "get_model", "get_grid", "get_frame",
+           "import_sql_select", "parse_setup", "parse_raw", "assign", "deep_copy", "get_model", "get_grid", "get_frame",
```
Comment (mmalohlava) [inline h2o-py/h2o/__init__.py]: Do we have some style check in Python?

**Comment by navdeep-G** [comment]
Gone through personal jenkins?

File: h2o-py/h2o/h2o.py [hunk 2]
```diff
@@ -1243,13 +1165,75 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
...
+    cluster().network_test()
+
+# Deprecated since 2016-08-04
+@deprecated("Deprecated, use ``h2o.cluster().timezone``.")
+def get_timezone():
+    """Deprecated."""
+    _check_connection()
+    return cluster().timezone
+
+# Deprecated since 2016-08-04
```
Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: Is there any 'annotation' for deprecated? It would help tracking

**Comment by mmalohlava** [comment]
Code looks ok for me. I pushed it for testing into my private jenkins.

File: h2o-py/h2o/__init__.py (hunk 1 above)
Comment (st-pasha) [inline h2o-py/h2o/__init__.py]: One can set up automatic linting in Intellij or Sublime, but we don't have any lint checks on commit (that's a Phabricator's feature).

**Comment by navdeep-G** [comment]
Cool. Looks good. Can merge after persona jenkins.

**Comment by mmalohlava** [comment]
My jenkins is almost green except the startup checks:

```
14:03:16 ================================ EXCEPTION INFO ================================
14:03:16 
14:03:16 [LOCAL FRAMES]
14:03:16 Omitted: imported modules, class declarations, __future__ features, None-valued
14:03:16 
14:03:16   Within <module>() line 33 in file h2o.init_test_HOQE-16.py:
14:03:16                    server: <h2o.backend.server.H2OLocalServer object at 0x7f94b920e5d0>
14:03:16                      conn: <H2OConnection to http://127.0.0.1:55330, no session>
14:03:16                      PORT: 55330
14:03:16 
14:03:16   Within __getattr__() line 74 in file ../../h2o/utils/backward_compatibility.py:
14:03:16                      self: <H2OConnection to http://127.0.0.1:55330, no session>
14:03:16                      item: cluster_is_up
14:03:16 
14:03:16 [STACKTRACE]
14:03:16 
14:03:16   File h2o.init_test_HOQE-16.py:
14:03:16                 <module>() #0033  cluster_up = conn.cluster_is_up()
14:03:16 
14:03:16   File ../../h2o/utils/backward_compatibility.py:
14:03:16              __getattr__() #0074  return getattr(self.__class__, item)```

http://mr-0xa1:8080/view/michal/job/michal_startup_checks/169/console

`cluster_is_up` is missing ?

**Comment by st-pasha** [comment]
Oh yes, looks like I missed it.

**Comment by st-pasha** [comment]
I made few fixes and pushed to my Jenkins once again

**Comment by st-pasha** [comment]
My jenkins is all green now, except for win7_pyunit_small, where 1 job failed with a DistributedException

**Comment by mmalohlava** [comment]
👍 (it is TN-4 failure - unstable DL model)

Looks good for me.

**Comment by st-pasha** [comment]
ok i pushed it to master

**Pull request closed without merging**